plugins/generator-1.18/forge-1.18/templates/tag.json.ftl

```diff
@@ -0,0 +1,29 @@
+<#include "mcitems.ftl">
+{
+    "replace": false,
+    "values": [
+      <#if data.type == "Items">
+          <#list w.filterBrokenReferences(data.items) as value>
+            "${mappedMCItemToIngameNameNoTags(value)}"<#if value?has_next>,</#if>
+          </#list>
+      <#elseif data.type == "Blocks">
+          <#list w.filterBrokenReferences(data.blocks) as value>
+            "${mappedMCItemToIngameNameNoTags(value)}"<#if value?has_next>,</#if>
+          </#list>
+      <#elseif data.type == "Functions">
+          <#list data.functions as value>
+            "${generator.getResourceLocationForModElement(value)}"
+            <#if value?has_next>,</#if>
+          </#list>
+      <#elseif data.type == "Entities">
+          <#list w.filterBrokenReferences(data.entities) as value>
+              <#if value.getUnmappedValue().startsWith("CUSTOM:")>
+                "${generator.getResourceLocationForModElement(value.getUnmappedValue()?replace("CUSTOM:", ""))}"
+              <#else>
+                "${generator.map(value.getUnmappedValue(), "entities", 0)}"

```

Comment:
Why is it now using mapping list 0 instead of 2?

Reply:
I took templates into the 1.17 data pack generator. I'll fix it.

Reply:
Yes, and please test your PRs..

Reply:
I made it in 2 minutes before going to a class... and I a bit forgot they don't have the same mappings. 😬😅

Reply:
Well you can make it before class, but then take time to test when you have time before opening PR :)